portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,16 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+
+.center {
```

Comment:
Is this selector being used for anything?

Reply:
No, it's just extra whitespace I made on accident when working with the CSS.

Reply:
If the .center selector isn't being used, can you delete it from the CSS file?
(If it is being used, can you delete the extra blank line above it?)